lib/membrane/rtp/session_bin.ex

```diff
@@ -66,13 +66,15 @@ defmodule Membrane.RTP.SessionBin do
 
   An extension will be spawned inside the bin under `{extension_name, ssrc}` name.
 
-  ### Currently supported RTP extensions are:
+  ### RTP plugin ships with the following extensions:
   * `Membrane.RTP.VAD`
   * `Membrane.RTP.TWCCReceiver`
+  * `Membrane.RTP.TWCCSender`
```

Comment:
Just an idea - might be considered outside the scope of this PR - why do we add `:twcc` & `:vad` keys if we also use unique atoms as a value? It should be enough to pass just the module atoms/structs, e.g. instead of
```elixir
[
  vad: %Mebrane.RTP.VAD{vad_id: 1, time_window: 1_000_000},
  twcc: %Mebrane.RTP.TWCCReceiver{twcc_id: 1, report_interval: Membrane.Time.milliseconds(250)}
]
```
we could just do:
```elixir
[
  %Mebrane.RTP.VAD{vad_id: 1, time_window: 1_000_000},
  %Mebrane.RTP.TWCCReceiver{twcc_id: 1, report_interval: Membrane.Time.milliseconds(250)}
]
```

Reply:
Looks like the `extension_name` is used for naming pipeline child corresponding to the given extension https://github.com/membraneframework/membrane_rtp_plugin/blob/master/lib/membrane/rtp/session_bin.ex#L434

and it is also allowed in `Membrane.RTP.Header.Extension.t`. I am not sure why :thinking: